docs/app/getting-started.md

```diff
@@ -21,17 +19,25 @@ A very simple [docker-compose.yml](/docker-compose.yml) has been included to sup
 
 3. If you are using an M1 mac, you will need to install postgres as well: `brew install postgresql` (The psycopg2-binary is built from source on M1 macs which requires the postgres executable to be present)
 
-4. You'll also need [Docker Desktop](https://www.docker.com/products/docker-desktop/)
+4. You'll also need [Docker Desktop](https://www.docker.com/products/docker-desktop/) installed and running.
 
 ## Run the application
 
-1. In your terminal, `cd` to the `app` directory of this repo.
-2. Make sure you have [Docker Desktop](https://www.docker.com/products/docker-desktop/) installed & running.
-3. Run `make setup-local` to install dependencies
-4. Run `make init start` to build the image and start the container.
-5. Navigate to `localhost:8080/docs` to access the Swagger UI.
-6. Run `make run-logs` to see the logs of the running API container
-7. Run `make stop` when you are done to delete the container.
+**Note:** Run everything from within the `/app` folder:
+
+1. Run `make init start` to build the image and start the container.
+2. Navigate to `localhost:8080/docs` to access the Swagger UI.
+3. Run `make run-logs` to see the logs of the running API container
+4. Run `make stop` when you are done to delete the container.
```

Comment:
This line is a bit misleading. Deleting might make someone think that the contents of their container will be removed when they run stop, but that isn't true as that's cached and/or stored in the volume.

Maybe we should adjust this to something like `.. when you are done to stop the container.`

Reply:
👍  will make this change!